From 3583818721f1808f427ad7e4c00a04013d3df5f1 Mon Sep 17 00:00:00 2001
From: Yehuda Yitschak <yehuday@marvell.com>
Date: Thu, 18 May 2017 15:12:52 +0300
Subject: [PATCH 2014/2241] defconfig: Enable generic uio driver in
 v8_lsp_defconfig

Enable generic platform uio driver (CONFIG_UIO_PDRV_GENIRQ), this is
used by user-space drivers in order to map regs and interrupts of kernel
drivers based on their dts information.

Change-Id: I3ffe231a0ba06249b286a03792b93115bb9944d4
Signed-off-by: Yehuda Yitschak <yehuday@marvell.com>
Signed-off-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-by: Haim Boot <hayim@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/44834
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/configs/mvebu_v8_lsp_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/mvebu_v8_lsp_defconfig b/arch/arm64/configs/mvebu_v8_lsp_defconfig
index b08ce48..ce7a998 100644
--- a/arch/arm64/configs/mvebu_v8_lsp_defconfig
+++ b/arch/arm64/configs/mvebu_v8_lsp_defconfig
@@ -218,6 +218,7 @@ CONFIG_MV_XOR_V2=y
 CONFIG_ASYNC_TX_DMA=y
 CONFIG_DMATEST=y
 CONFIG_UIO=y
+CONFIG_UIO_PDRV_GENIRQ=m
 CONFIG_UIO_PCI_EP=y
 CONFIG_VFIO=y
 CONFIG_VFIO_PCI=y
-- 
2.7.4

